### PR TITLE
Create trim.md

### PR DESCRIPTION
**Pull request by Captain-Arnab**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+Title: 'trim()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Strip whitespace (or other characters) from the beginning and end of a string' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:3]:
```suggestion
Description: 'Strips whitespace (or other characters) from the beginning and end of a string.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Web Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'String'
+  - 'Functions'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-php'
+  - 'path/computer-science'
+---
+
+`trim()` returns a string with whitespace stripped from the beginning and end of string.
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:16]:
```suggestion
The **`trim()`** function returns a string with whitespace stripped from the beginning and end of string.
```

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+The string that will be trimmed.
+
+characters
+Optionally, the stripped characters can also be specified using the characters parameter. Simply list all characters that you want to be stripped. With .. you can specify a range of characters.
+
+## Syntax
+
+`$trimmed = trim($text, " \t.");`
+
+## Codebyte Example (if applicable)
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:34]:
```suggestion
## Codebyte Example
```

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+```
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:36]: This block needs to start with ` ```codebyte/php `.
It also needs a sentence preceding it, describing what it does.

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  - 'String'
+  - 'Functions'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-php'
+  - 'path/computer-science'
+---
+
+`trim()` returns a string with whitespace stripped from the beginning and end of string.
+
+## Pseudo Code:
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:18]:
```suggestion
## Syntax
```

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+`trim(string $string, string $characters = " \n\r\t\v\x00"): string`
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:20]: This should be in a ` ```pseudo ` code block, not single backticks.

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+Parameters:
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:22]: Rewrite this Parameters section by removing the line of dashes and organizing the parameters in a bulleted list like this:

- `string`: The string that will be trimmed.
- `characters`: Optionally, the stripped characters can also be specified using the characters parameter.

Also, rewrite to remove the use of the "you" pronoun.

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+------------------------------------------------------
+string
+The string that will be trimmed.
+
+characters
+Optionally, the stripped characters can also be specified using the characters parameter. Simply list all characters that you want to be stripped. With .. you can specify a range of characters.
+
+## Syntax
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/trim/trim.md:30]: Remove this section.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Captain-Arnab I did a first review and left some comments. There may be more after you address these.

**Comment by SSwiniarski** [comment]
@Captain-Arnab It's been over two weeks, do you need assistance addressing these changes?

**Comment by Captain-Arnab** [comment]
@SSwiniarski I have committed the changes. Please check those.

**Comment by SSwiniarski** [comment]
@Captain-Arnab I don't see your changes. Did you make commits to the branch this PR is based on?

**Comment by SSwiniarski** [comment]
@Captain-Arnab your changes never made it to this PR, do you need assistance?

**Comment by SSwiniarski** [comment]
@Captain-Arnab It's been another two weeks, are you still working on this PR?

**Review by SSwiniarski — APPROVED** [review]
@Captain-Arnab because of the inactivity, I addressed the concerns myself so we can move on to a second review.

File: content/php/concepts/string-functions/terms/trim/trim.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+```pseudo
+trim($string, $characters)
+```
+
+Parameters:
+
+- `$string`: The string that will be trimmed.
+- `$characters`: Optionally, the stripped characters can be specified using the characters parameter. A range of characters can be specified with `..`. By default, it will strip:
+  - Spaces `" "` \n\r\t\v\x00"
```
Comment (yangc95) [inline content/php/concepts/string-functions/terms/trim/trim.md:28]:
```suggestion
  - Spaces `" "`
```

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks for your contribution @Captain-Arnab 👍

Live link: https://www.codecademy.com/resources/docs/php/string-functions/trim

cc: @SSwiniarski

**Pull request merged**